src/screens/MainScreen.js

```diff
@@ -1,101 +1,110 @@
-import { StatusBar } from 'expo-status-bar';
-import { useState, useEffect } from 'react'
-import { StyleSheet, Text, View, Button, TouchableOpacity, Dimensions } from 'react-native';
-import useAuthentication from '../utils/useAuthentication';
-import { getStringValue, setStringValue, removeValue } from '../utils/asyncStorage';
-import { FontAwesome5, Feather, EvilIcons } from '@expo/vector-icons';
-import FadeInAnimatedText from '../components/FadeInAnimatedText';
-import RevealMessage from '../components/RevealMessage';
-import Header from '../components/Header';
-import { globalStyle, colors } from '../styles/styles';
+import { StatusBar } from "expo-status-bar";
+import { useState, useEffect } from "react";
+import { StyleSheet, Text, View, Button, TouchableOpacity } from "react-native";
+import useAuthentication from "../utils/useAuthentication";
+import {
+  getStringValue,
+  setStringValue,
+  removeValue,
+} from "../utils/asyncStorage";
+import { FontAwesome5, EvilIcons } from "@expo/vector-icons";
+import FadeInAnimatedText from "../components/FadeInAnimatedText";
+import RevealMessage from "../components/RevealMessage";
+import Header from "../components/Header";
+import { globalStyle, colors } from "../styles/styles";
 
 /** Screen where messages are seen. */
-const MainScreen = ( {navigation} ) => {
-  const [lastMessageTime, setLastMessageTime] = useState()
-  const { user } = useAuthentication();
```

Comment:
not sure if this an issue

Reply:
user was not being used

Reply:
but I was assuming we would need user on all screens at some point